policyengine_canada/parameters/gov/provinces/nb/tax/income/credits/tuition_amount/phase_out_rate.yaml

```diff
@@ -0,0 +1,34 @@
+description: New Brunswick phases taxable income out for tuition amount at this rate, based on taxable income.
```

Comment:
@PavelMakarchuk Hi Pavel, I got a question here. I double checked the [form](https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5004-c/5004-c-22e.pdf#page=1), the following brackets is used to calculate the tax on taxable income, shall we use "phase out" here? Since "phase out" is about with income increases and then credits decrease, this part doesn't directly relate to the credit, it's one step of the calculation.